h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1272,11 +1275,14 @@ else if (s.equals("-proxy")) {
       } else if (s.equals("-hivePrincipal")) {
         i++; if (i >= args.length) { usage (); }
         hivePrincipal = args[i];
-      } else if (s.equals("-refreshTokens")) {
-        refreshTokens = true;
+      } else if (s.equals("-refreshTokens") || // for backwards compatibility 
```

Comment:
We cannot really deprecate it because Steam needs to be able to support all H2O versions. This means Steam will likely keep using "-refreshTokens" with semantics "refresh hive tokens" (and we shouldn't generate a warning for our own software).

Could change in the future, for now I think leave as-is.

Reply:
ok, np